.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 1
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 1
```

Comment:
I'm assuming 0 days isn't an option.

Reply:
It's not clearly noted in the documentation but IIUC the stale bot implementation will accept 0 days, it will then identify pr's updated before the current time.